features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -117,12 +117,10 @@ CREATE TABLE IDN_OAUTH2_ACCESS_TOKEN_SCOPE (
             FOREIGN KEY (TOKEN_ID) REFERENCES IDN_OAUTH2_ACCESS_TOKEN(TOKEN_ID) ON DELETE CASCADE)
 /
 CREATE TABLE IDN_OAUTH2_SCOPE (
-            SCOPE_ID INTEGER NOT NULL,
-            SCOPE_KEY VARCHAR (100) NOT NULL,
-            NAME VARCHAR (255) NULL,
-            DESCRIPTION VARCHAR (512) NULL,
-            TENANT_ID INTEGER DEFAULT 0 NOT NULL,
-            ROLES VARCHAR (500) NULL,
+            SCOPE_ID VARCHAR(1024) NOT NULL UNIQUE,
+            NAME VARCHAR(255) NOT NULL,
+            DESCRIPTION VARCHAR(512) NOT NULL,
+            TENANT_ID INTEGER NOT NULL DEFAULT 0,
```

Comment:
I think better to have -1234 as the default value here, if it is not a problem with migration.